sdk/python/foundation-models/benchmarking/evaluating_oai_models.ipynb

```diff
@@ -0,0 +1,199 @@
+    "\n",
+    "## Prerequistes\n",
+    "- An Azure account with an active subscription - [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)\n",
+    "- An Azure ML workspace with compute cluster - [Configure workspace](../../configuration.ipynb)\n",
+    "- An OpenAI model deployment in the Azure OpenAI service - [Create and deploy an Azure OpenAI Service resource](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)\n",
+    "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section\n",
+    "- A python environment with [mlflow](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-mlflow-configure-tracking?view=azureml-api-2&tabs=python%2Cmlflow) for retrieving eval metrics\n",
+    "\n",
+    "## Configuring a Workspace connection for Azure OpenAI access\n",
+    "You will use a Workspace connection to securely store OpenAI access keys. Follow the directions for [creating an AOAI connection in the AzureML studio UI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/get-started-prompt-flow?view=azureml-api-2#connection).\n",
+    "\n",
+    "## Configuring and running an evaluation pipeline\n",
+    "Please set global values in the following cell for your AzureML Workspace, the Azure OpenAI endpoint you want to call, the name of connection you created in the previous step, and the name of the eval you want to run.\n",
+    "\n",
+    "Supported evals are the following: `boolq`, `gsm8k`, `hellaswag`, `human_eval`, `mmlu_humanities`, `mmlu_other`, `mmlu_social_sciences`, `mmlu_stem`, `openbookqa`, `piqa`, `social_iqa`, `winogrande`.\n",
+    "\n",
+    "You can also set the sample ratio, the fraction of the selected dataset to run for the eval.\n",
+    "\n",
+    "**Warning**: Many datasets contain thousands of examples which can lead to high endpoint usage costs. We advise starting with a small sample ratio (e.g., 1%) to verify the pipeline and then increasing the ratio if desired. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ce3f8a4-ce03-462b-89fa-19cac827c30c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# AzureML settings - please fill in your values\n",
+    "subscription_id = \"<Azure subscription ID>\"\n",
+    "resource_group = \"<Resource group>\"\n",
+    "workspace_name = \"<Workspace name\"\n",
+    "experiment_name = \"<Experiment name>\"\n",
+    "compute_name = \"<AzureML Compute name>\"\n",
+    "\n",
+    "# Eval to run - you can change this to any of the 12 supported eval names\n",
+    "eval_name = \"mmlu_humanities\"\n",
+    "\n",
+    "# AOAI endpoint URL, ex: https://resource-name.openai.azure.com/openai/deployments/gpt-4/chat/completions?api-version=2023-07-01-preview\n",
+    "aoia_endpoint_url = \"<Endpoint URL>\"\n",
```

Comment:
There is typo in "aoia_endpoint_url". It should be "aoai" instead of "aoai".

Reply:
Fixed